docs_user/modules/openstack-edpm_adoption.adoc

```diff
@@ -275,6 +275,7 @@ spec:
     - nova-compute-extraconfig
     - ovn
     - neutron-metadata
+    - telemetry
```

Comment:
we also need to update the test: https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/tests/roles/dataplane_adoption/tasks/main.yaml#L155